packages/tocco-ui/src/Menu/Menu.js

```diff
@@ -0,0 +1,51 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import styled from 'styled-components'
+
+import {stylingLook} from '../utilStyles'
+
+const StyledMenu = styled.ul`
+  && {
```

Comment:
(what does `&&` stand for?)

Reply:
"&&" doubles css class hash in stylesheet to increase specificity (e.g. ".abcd.abcd {}"). Amount of ampersanda is multiplication factor for hash.

see also https://www.styled-components.com/docs/faqs#how-can-i-override-styles-with-higher-specificity

As long as tocco-theme is active we need that to ensure correct styling (due line 7 in packages/tocco-theme/src/ToccoTheme/theme.scss and the fact that loading order can not be guaranteed). 

If our flows goes implemented in third party website, it is advisable to have a certain level of specifity to ensure tocco-client styling. E.g. an agency integrates a flow in Bootstrap Panel. Without increaseing specifity those style would leak into our flows. It is not yet sure what the best factor is.